client/reducers/SearchReducer.js

```diff
@@ -0,0 +1,11 @@
+// import findIndex from 'lodash/findIndex';
+
+export default (state = [], action = {}) => {
+  switch (action.type) {
+    case 'IS_SEARCH': {
+      return action.searchPayload;
```

Comment:
The purpose of reducers is to return an altered **copy** of the **current state** not what the action returns
please refer to: http://redux.js.org/docs/basics/Reducers.html#handling-more-actions